FlatMate/View/OnboardingPageView.swift

```diff
@@ -50,7 +50,7 @@ struct LandingPageView: View {
                         if self.currentStep > 0 {
                             self.currentStep -= 1
                         }
-                    })
+                    }, type: .standard)
```

Comment:
Can we make `.standard` the default? (i.e. if no type is specified it is standard)

Reply:
Good point.